src/derivation.js

```diff
@@ -418,21 +418,42 @@ export const getDerivationModesForCurrency = (
     all.push("legacy_on_segwit");
     all.push("legacy_on_native_segwit");
   }
-  if (!disableBIP44[currency.id]) {
-    all.push("");
-  }
   if (currency.supportsNativeSegwit) {
     all.push("native_segwit");
   }
   if (currency.supportsSegwit) {
     all.push("segwit");
   }
+  if (!disableBIP44[currency.id]) {
+    all.push("");
+  }
```

Comment:
I don't think we should reorder this. Or maybe we need to reorder many of them, but right now the idea was to scan the most legacy first and least legacy at the end.

Reply:
Maybe it's ok to reverse it all because indeed native segwit and segwit ordering was already not really consistant.

Reply:
In short my question is: is this change necessary / why was it needed? Let's discuss

Reply:
without this when we first start the scan of the accounts we get an error on the front end side while we retrieve the other types of accounts + the newly created accounts ordering is messed up at the end and we show the legacy first which is not what we want :shrug: i remember we already switched the native and segwit order once i though this might be ok

Reply:
ok. i'm ok with it.
**on a more long term discussion, i genuinely wonder if we should put all the "advanced/custom" searches actually AFTER these, so we truely have a reversed order** (from most preferable to less likely).

I actually like the other way (most likely to less likely), because it will make the UX perceived as faster anyway... let's keep this, but we need to keep the question for the future.